src/openrct2/interface/Window_internal.h

```diff
@@ -71,6 +71,7 @@ struct rct_window
     {
         int16_t picked_peep_frame; // Animation frame of picked peep in staff window and guest window
         int16_t var_492;
+        uint32_t NumGuests;
```

Comment:
I can't see this being used anymore after my change.

Reply:
You're right. I have removed it.